packages/wk-lib-playground/package.json

```diff
@@ -18,9 +19,7 @@
   "author": "bnaya eshet",
   "license": "MIT",
   "types": "dist/index.d.ts",
```

Comment:
Evidently consumers are looking for types with the "typings" field in package.json.   Might be that types is not necessary (I may have to update my blog entry) 😄